agent/api/task.go

```diff
@@ -72,11 +72,11 @@ type Task struct {
 	Containers []*Container
 	// Volumes are the volumes for the task
 	Volumes []TaskVolume `json:"volumes"`
-	// VCPULimit is a task-level limit for compute resources. A value of 1 means that
+	// Cpu is a task-level limit for compute resources. A value of 1 means that
 	// the task may access 100% of 1 vCPU on the instance
-	VCPULimit float64 `json:"CPULimit,omitempty"`
-	// MemoryLimit is a task-level limit for memory resources in bytes
-	MemoryLimit int64 `json:"Memory,omitempty"`
+	Cpu float64 `json:"CPULimit,omitempty"`
```

Comment:
I think `CPU` should also work. Can you test these fields in this [file](https://github.com/aws/amazon-ecs-agent/blob/master/agent/acs/handler/acs_handler_test.go#L53)?